configs/pykeen/pykeen.yaml

```diff
@@ -0,0 +1,59 @@
+target_repository:
+  HTTPS: https://github.com/pykeen/pykeen.git
+  # OPTIONAL, for some protected repository
+  # username: my-nick
+  # OPTIONAL, paired with the username
+  # password: dont-tell-anyone
+  # OPTIONAL, overrides the user/pass
+  # token: authentication-token
+  # OPTIONAL, checkout a particular branch or a tag
+  # TODO: change once https://github.com/pykeen/pykeen/pull/930 is merged
+  checkout: pyl-tests
+  # checkout: master
+  # OPTIONAL, define installing package extras
+  # install_extras: all
+  # OPTIONAL, install additional requirements from a file
+  # requirements_file: requirements.txt
+  # copy some  test from the target repository
+  copy_tests:
+    - tests/test_lightning.py
+
+contact:
+  slack:
+    - U03DAAVMLSH
+  email: []
+
+# OPTIONAL, running before installing your project
+# WARNING: this is custom for given OS, we do not manage any inter OS compatibility
+#before_install:
+#  - pip install Cython==3.56
+
+# OPTIONAL, if any installation require some env. variables
+#env:
+#  HOROVOD_BUILD_ARCH_FLAGS: "-mfma"
+#  # HOROVOD_WITHOUT_MXNET: 1
+#  # HOROVOD_WITHOUT_TENSORFLOW: 1
+
+dependencies:
+  - name: pytorch-lightning
+    HTTPS: https://github.com/PyTorchLightning/pytorch-lightning.git
+    checkout: release/latest
+    # install_extras: all
+#  - name: Cython
+#    checkout: 0.29.24
+
+# OPTIONAL, running before installing your project
+# WARNING: this is custom for given OS, we do not manage any inter OS compatibility
+#before_test:
+#  - wget https://pl-flash-data.s3.amazonaws.com/titanic.zip
+
+testing:
+  dirs:
+    - tests/
+#  # OPTIONAL, additional pytest arguments
+#  pytest_args: --strict
+
+runtimes:
+  - {os: "ubuntu-20.04", python-version: "3.10"}
+  - {os: "macOS-10.15", python-version: "3.9"}
+  - {os: "windows-2019", python-version: "3.8"}
```

Comment:
In [PyKEEN](https://github.com/pykeen/pykeen/blob/e26fb956c96528c0d12c63cf6de05b0cf4bf71c4/.github/workflows/tests.yml#L44-L46) we are testing with Python 3.8 and 3.10, which are the minimum and maximum Python version we officially support. We test for both, [Ubuntu](https://github.com/pykeen/pykeen/blob/e26fb956c96528c0d12c63cf6de05b0cf4bf71c4/.github/workflows/tests.yml#L72) and [Windows](https://github.com/pykeen/pykeen/blob/e26fb956c96528c0d12c63cf6de05b0cf4bf71c4/.github/workflows/tests.yml#L92-L93), but do not have specific tests for macOS.

If you have any suggestions here, feel free to share them. The main reason we are interested in integrating with [PyTorchLightning/ecosystem-ci](https://github.com/PyTorchLightning/ecosystem-ci/) is the possibility to run tests with GPUs, which we at PyKEEN do not have available for continous testing.

Reply:
that is fine we can test only Linux and Win or just Ubuntu :otter: